triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -43,24 +55,60 @@ object TriggerServiceFixture {
       mat: Materializer,
       aesf: ExecutionSequencerFactory,
       ec: ExecutionContext): Future[A] = {
+    // Launch a toxiproxy instance . Wait on it to be ready to accept
+    // connections.
+    val toxiExe = BazelRunfiles.rlocation("external/toxiproxy_nix/bin/toxiproxy-cmd")
+    val toxiProxyPort = findFreePort()
+    val toxiCmd = toxiExe + " --port " + toxiProxyPort.toString
```

Comment:
I suggest using a `Seq` here. `Process.apply` accepts that too and you don't need to worry about shell parsing.

```suggestion
    val toxiCmd = Seq(toxiExe, "--port", toxiProxyPort.toString)
```